docs/data/material/getting-started/templates/dashboard/components/ChartUserByCountry.tsx

```diff
@@ -0,0 +1,174 @@
+import * as React from 'react';
+import { PieChart } from '@mui/x-charts/PieChart';
+import { useDrawingArea } from '@mui/x-charts/hooks';
+import { styled } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import LinearProgress, { linearProgressClasses } from '@mui/material/LinearProgress';
+
+import {
+  IndiaFlag,
+  UsaFlag,
+  BrazilFlag,
+  GlobeFlag,
+} from '../internals/components/CustomIcons';
+
+const data = [
+  { label: 'India', value: 50000 },
+  { label: 'USA', value: 35000 },
+  { label: 'Brazil', value: 10000 },
+  { label: 'Other', value: 5000 },
+];
+
+const countries = [
+  {
+    name: 'India',
+    value: 50,
+    flag: <IndiaFlag />,
+    color: 'hsl(220, 25%, 65%)',
+  },
+  {
+    name: 'USA',
+    value: 35,
+    flag: <UsaFlag />,
+    color: 'hsl(220, 25%, 45%)',
+  },
+  {
+    name: 'Brazil',
+    value: 10,
+    flag: <BrazilFlag />,
+    color: 'hsl(220, 25%, 30%)',
+  },
+  {
+    name: 'Other',
+    value: 5,
+    flag: <GlobeFlag />,
+    color: 'hsl(220, 25%, 20%)',
+  },
+];
+
+interface StyledTextProps {
+  variant: 'primary' | 'secondary';
+}
+
+const StyledText = styled('text', {
+  shouldForwardProp: (prop) => prop !== 'variant',
+})<StyledTextProps>(({ theme, variant }) => ({
+  textAnchor: 'middle',
+  dominantBaseline: 'central',
+  fill: theme.palette.text.secondary,
+  fontSize:
+    variant === 'primary'
+      ? theme.typography.h5.fontSize
+      : theme.typography.body2.fontSize,
+  fontWeight:
+    variant === 'primary'
+      ? theme.typography.h5.fontWeight
+      : theme.typography.body2.fontWeight,
+}));
+
+interface PieCenterLabelProps {
+  primaryText: string;
+  secondaryText: string;
+}
+
+function PieCenterLabel({ primaryText, secondaryText }: PieCenterLabelProps) {
+  const { width, height, left, top } = useDrawingArea();
+  const primaryY = top + height / 2 - 10;
+  const secondaryY = primaryY + 24;
+
+  return (
+    <React.Fragment>
+      <StyledText variant="primary" x={left + width / 2} y={primaryY}>
+        {primaryText}
+      </StyledText>
+      <StyledText variant="secondary" x={left + width / 2} y={secondaryY}>
+        {secondaryText}
+      </StyledText>
+    </React.Fragment>
+  );
+}
+
+const colors = [
+  'hsl(220, 25%, 65%)',
+  'hsl(220, 25%, 45%)',
+  'hsl(220, 25%, 30%)',
+  'hsl(220, 25%, 20%)',
+];
+
+export default function ChartUserByCountry() {
```

Comment:
This component is flaky: https://app.argos-ci.com/mui/material-ui/builds/28511/92821674.

But it shouldn't be in the visual regressions test in the first place, since the parent one is into it. To remove.

Reply:
But also, it seems that we need @alexfauquette patch in https://github.com/mui/mui-x/pull/9926/files#diff-26370639d5cddf1b1016c2de2ceb142f773f358cf916cbc637bdd327be854495

Reply:
Argos has been quick patch, and a refinement PR is on the way #42537